proto/spacemesh/v1/global_state_types.proto

```diff
@@ -50,22 +50,42 @@ message AccountDataQueryRequest {
     uint32 offset = 3; // return results from offset
 }
 
+message ExecutionError {
```

Comment:
We have standard error/status types already, with the same data elements, maybe we don't need to introduce another one here. See "google.golang.org/grpc/codes", "google.golang.org/grpc/status"